resources/metadata/npmExecuteScripts.yaml

```diff
@@ -175,3 +175,6 @@ spec:
   containers:
     - name: node
       image: node:lts-buster
+      options:
```

Comment:
This change is needed as Azure agents need root permissions to install globally.
We will need to inform users ahead of this change so that if they don't need it , it could be overridden manually using dockerOptions parameter